src/goto-instrument/cover_basic_blocks.cpp

```diff
@@ -228,3 +160,46 @@ void cover_basic_blockst::update_covered_lines(block_infot &block_info)
   std::string covered_lines = format_number_range(line_list);
   block_info.source_location.set_basic_block_covered_lines(covered_lines);
 }
+
+cover_basic_blocks_javat::cover_basic_blocks_javat(
+  const goto_programt &_goto_program)
+{
+  forall_goto_program_instructions(it, _goto_program)
+  {
+    const auto &bytecode_index = it->source_location.get_java_bytecode_index();
+    if(!index_to_block.count(bytecode_index))
+    {
+      block_infos.push_back(it);
```

Comment:
Where are the covered_lines set?

Reply:
They are not. Because I don't see how instructions with same bytecode index can correspond to different lines?

Reply:
They must be set nonetheless in order not to change the output format.

Reply:
Oh right, I thought this was the same as get_line. This should be fixed in the last fixup commit https://github.com/diffblue/cbmc/pull/1830/commits/48fc65ceb2d5ec425e074e37a809237f4fa85a13